src/css/custom.css

```diff
@@ -491,3 +524,129 @@ a.table-of-contents__link.toc-highlight > span.badge {
 .tabs-hidden {
   display: none;
 }
+
```

Comment:
blocking, I think:

The [release notes pages](https://preview.docs.camunda.cloud/pr-4324/docs/next/reference/release-notes/860/) are a unique usage of the badges. With these heading size changes, they look a little silly. On the left is without these changes, on the right is with these changes: 

<img width="1739" alt="image" src="https://github.com/user-attachments/assets/a5070c07-63ab-4bad-84ac-8af499c42749">

I _think_ we would want the text in these badges to be a little smaller, but maybe I'm wrong.

Reply:
Thanks for spotting - I wasn't sure where all these badge versions were used - I've amended these ones on the release notes page to have smaller font, and fixed padding to be better 👍

Reply:
What do you think about adding `vertical-align:middle` to the `h3 .badge--long` selector, to nudge the vertical placement of the badges in this context? 

Thus changing from this: 

<img width="443" alt="image" src="https://github.com/user-attachments/assets/50c31ac8-d415-4643-80d0-a7fafaa6c0a8">

to this:

<img width="550" alt="image" src="https://github.com/user-attachments/assets/490b67b1-6144-448a-a831-13bb341dd899">

It's not perfect, but it does seem to bring them closer to in-line with the heading.